packages/widgets-internal/components/NetworkFilter/NetworkFilter.tsx

```diff
@@ -9,7 +9,9 @@ export interface INetworkProps {
   onChange: (value: INetworkProps["value"]) => void;
 }
 
-const Container = styled.div<{ isShow: boolean }>`
+const Container = styled.div.withConfig({
```

Comment:
use `$props` style for styled only props, which will not pass to dom

https://styled-components.com/docs/api#using-custom-props

Reply:
done